run_dir/design/flowcells.html

```diff
@@ -262,13 +262,26 @@ <h1><span id="illumina_page_title">Illumina flowcells</span>
       "paging":false,
       "info":false,
       "order": [[ 1, "desc" ]],
+      colReorder: true,
       dom: 'Bfrti',
       buttons: [
         { extend: 'copy', className: 'btn btn-outline-dark mb-3' },
         { extend: 'excel', className: 'btn btn-outline-dark mb-3' }
       ]
     });
 
+    //Keep markdown format when dragging around the columns
+    table.on( 'column-reorder', function() {
+        $(".running-note-card > .card-body").each(function(i){
+            $(this).html(make_markdown($(this).text()));
+            $('.fillbadgecolour').html(function(){
```

Comment:
This function `$('.fillbadgecolour').html(function(){` should be outside the `$(".running-note-card > .card-body").each(function(i){` function.

Reply:
I have set sanity checks on myself, works now on stage 🤷🏻‍♀️